sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -622,6 +840,22 @@ public virtual async Task<Response> CloseGroupConnectionsAsync(string group, IEn
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="group"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="group"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call CloseGroupConnections with required parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = client.CloseGroupConnections("group");
+        /// </code>
+        /// This sample shows how to call CloseGroupConnections with all parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = client.CloseGroupConnections("group", null, null);
```

Comment:
I think it would be better if we don't pass nulls in the examples, unless this is a common scenario.